### PR TITLE
DevEx: Remove extra comments, add string length limitations

### PR DESCRIPTION
**Pull request by ericsorenson**

### CODE REVIEW

File: shared/src/business/entities/cases/PublicCase.js [hunk 1]
```diff
@@ -55,27 +55,29 @@ function PublicCase(rawCase, { applicationContext }) {
...
     .uuid({
       version: ['uuidv4'],
     })
     .optional(),
   createdAt: joiStrictTimestamp.optional(),
-  docketNumber: joi.string().optional(),
-  docketNumberSuffix: joi.string().allow(null).optional(),
+  docketNumber: joi.string().max(500).optional(), // TODO: regex
+  docketNumberSuffix: joi.string().max(2).allow(null).optional(), // TODO: enum
   isSealed: joi.boolean(),
```
Comment (rachelschneiderman) [inline shared/src/business/entities/cases/PublicCase.js:68]: Should this be marked as optional?

File: shared/src/tools/court-issued-event-codes.csv [hunk 2]
```diff
@@ -10,7 +10,7 @@ OASL,"ORDER that the letters ""SL"" are added to the Dkt. No. *ANY*","OASL - Ord
 OAW,"ORDER that the letter ""W"" is added to the Dkt. No. *ANYTHING*","OAW - Order that the letter ""W"" is added to the Docket number ",Type C,,,,[Docket number],,"Order that the letter ""W"" is added to the Docket number [Docket number]",Served
 OAX,"ORDER that the letter ""X"" is added to the Dkt. No. *ANYTHING*","OAX - Order that the letter ""X"" is added to the Docket number ",Type C,,,,[Docket number],,"Order that the letter ""X"" is added to the Docket number [Docket number]",Served
 OCA,ORDER that caption of case is amended. *ANYTHING*,OCA - Order that caption of case is amended ,Type A ,[Anything],,,,,Order that caption of case is amended [Anything],Served
```
Comment (rachelschneiderman) [inline shared/src/tools/court-issued-event-codes.csv:12]: Is there an extra comma here?

File: shared/src/tools/court-issued-event-codes.csv [hunk 3]
```diff
@@ -35,10 +35,10 @@ OSC,ORDER *ANYTHING*,OSC - Order ,Type A ,[Anything],,,,,Order [Anything],Served
 OSCP,"ORDER petr(s) by *DATE* show cause why ""S"" should not be removed *ANYTHING*","OSCP - Order petr(s) to show cause why ""S"" should not be removed ",Type D,[Anything],,[Date],,,"OSCP - Order petr(s) by [Date] to show cause why ""S"" should not be removed [Anything]",Served
 OST,ORDER of Service of Transcript (Bench Opinion) *ANYTHING*,OST - Order of Service of Transcript (Bench Opinion),Type A ,[Anything],,,,,Order of Service of Transcript (Bench Opinion) [Anything],Served
```
Comment (rachelschneiderman) [inline shared/src/tools/court-issued-event-codes.csv:36]: Is this an extra comma as well?

**Review by rachelschneiderman — APPROVED** [review]

File: shared/src/tools/court-issued-event-codes.csv (hunk 2 above)
Comment (sutt0n) [reply shared/src/tools/court-issued-event-codes.csv:12]: I _think_ anything that's within a double-quote and has a comma _might be_ an extra comma.

**Comment by sutt0n** [comment]
With these refactors and changing these documentTypes, will we see any data discrepancies in deployed envs? Is there a migration needed?

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**